### ISSUE TITLE
SA1025 incorrectly reported in several contexts

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#259]
The following code results in warning SA1025 being reported for the use of multiple spaces in a row, even though the spaces are within the comment text.

``` csharp
// Examples are:
//   1. Foo
//   2. Bar
```

The following code results in warning SA1025, even though the spaces follow a comma (which is allowed).

``` csharp
int[] data = { 3,  4 };
```

The following code results in warning SA1025, even though the spaces follow a semicolon (which is allowed).

``` csharp
for (;  ;)
{
}
```
